docs/client_reference.rst

```diff
@@ -0,0 +1,261 @@
+.. _aiohttp-client-reference:
+
+HTTP Client Reference
+=====================
+
+.. highlight:: python
+
+.. module:: aiohttp.web
+
+
+.. _aiohttp-web-request:
+
+
+request function
+----------------
+
+.. py:function:: request(method, url, *, params=None, data=None, headers=None, cookies=None, files=None, auth=None, allow_redirects=True, max_redirects=10, encoding='utf-8', version=HttpVersion(major=1, minor=1), compress=None, chunked=None, expect100=False, connector=None, loop=None, read_until_eof=True, request_class=None, response_class=None, test=None)
+   :module: aiohttp.client
+
+   Performs an asyncronous http request. Returns a response object.
+
+   The function is a :ref:`coroutine <coroutine>`.
+
+   :param str method: HTTP method
+
+   :param str url: Request URL
+
+   :param params: Dictionary or bytes to be sent in the query
```

Comment:
Actually, only dict is supported. For bytes you'll get an error [here](https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/client.py#L442)